ee/clickhouse/sql/funnels/funnel_trend.py

```diff
@@ -0,0 +1,43 @@
+FUNNEL_TREND_SQL = """
+select day_start,
+    sum(completed) as total_completed_funnels,
+    sum(total) as all_funnels_entries,
+    cohort
+from (
+    {funnel_trend_null_sql}
+    left outer join (
+        select countIf(max_step={step_count}) as completed,
+            {interval_method}(when) as start,
+            count(1) as total,
+            groupArray(person_id) as cohort
+        from (
+            SELECT person_id, max_step, when
+            FROM (
+                SELECT
+                    pid.person_id,
+                    windowFunnel({within_time})(toUInt64(toUnixTimestamp64Milli(timestamp)),
+                        {steps}
+                    ) as max_step,
+                    max(timestamp) as when
+                FROM events
+                JOIN (
+                    SELECT person_id, distinct_id
+                      FROM ({latest_distinct_id_sql})
+                     WHERE team_id = {team_id}
+                ) as pid
+                ON pid.distinct_id = events.distinct_id
+                WHERE team_id = {team_id}
```

Comment:
NIT: on all other queries we treat team_id as a parameter so it will be %(team_id)s to adhere to query parameters when possible rather than string interpolation. This shouldn't really matter because team_id can't be a user input so there shouldn't be SQL Injection risk here but just a note on consistency